tests/property/test_verify.py

```diff
@@ -121,7 +120,7 @@ def test_verify_decryption(self):
         self.assertTrue(verification.verified)
 
     @settings(
-        deadline=timedelta(milliseconds=2000),
+        deadline=timedelta(milliseconds=10000),
```

Comment:
This is a significant bump of time. Does this need to be 5 times larger?

Reply:
I am not sure about time, saw 10000 ms on other test using same hypothesis.
https://github.com/microsoft/electionguard-python/blob/37b55504b62c07cd6083ddcbebb8c6af84e574e6/tests/property/test_tally.py#L39
and saw a test failed once on the action because it exceeds the deadline of 2000.00ms